### ISSUE TITLE
Options are not passed into the plugin

### ISSUE THREAD

**Issue by RIP21** [codemod-js/codemod#686]
I created dummy plugin that does nothing but `console.log` of options.
And if I run it like that

```
codemod -p ./index.js -o index=@codemod.json --printer prettier /src
```

with config like that

```
{ "test": true }
```

Or like that

```
codemod -p ./index.js -o index='{"test":true}' --printer prettier /src
```

It outputs `{}` into the console.

I tried to run it with name in the plugin like `-o name-of-the-plugin=@codemod.json` with no result 

That makes any codemod that requires options to be unusable.

**Comment by RIP21** [comment]
Ok. I was partially lying about creating a dummy codegen, cause I checked it on my https://github.com/RIP21/import-move-codemod and it failed as I have options check before doing anything. But there is definitely something is wrong.

When I actually ran a dummy plugin like (made of compiled sources, don't judge me)
```js
'use strict'
var helper_plugin_utils_1 = require('@babel/helper-plugin-utils')
var core_1 = require('@babel/core')
var plugin_syntax_typescript_1 = require('@babel/plugin-syntax-typescript')

exports.default = helper_plugin_utils_1.declare(function (_, options) {
  console.log(options)
  return {
    name: "import-move-codemod",
    inherits: plugin_syntax_typescript_1.default,
    visitor: {
      Program: function (path) {
        return
      },
    },
  }
})
```

Against 1 file like that (note that I edited sources of `import-move-codemod` output file directly in `node_modules/import-move-codemod`)
```
codemod -p import-move-codemod -o import-move-codemod=@codemod.json --printer prettier packages/ui/src/index.tsx
```
 I've got into console
```
{}
{ "test": true }
packages/ui/src/index.tsx
1 file(s), 0 modified, 0 errors
```
This means that for some reason it called plugin once without any options, and then with proper options as expected.

I sure can remove checks on the wrong input, but I think it should be fixed.

**Comment by eventualbuddha** [comment]
The extra call comes from [here](https://github.com/codemod-js/codemod/blob/a750ec6cf45cd87454c35687e107c98e35359f45/packages/cli/src/Config.ts#L23), which is done because codemod is trying to get the name of the plugin if it declares one. Honestly, I didn't even realize that babel passed the plugin options at the top level. According the babel handbook, [you can check options at each visitor method](https://github.com/jamiebuilds/babel-handbook/blob/master/translations/en/plugin-handbook.md#toc-plugin-options). To do validation of the plugin, you could check in a `Program` visitor. That said, [it looks like the canonical way to do it is the way you're trying to do it](https://github.com/babel/babel/blob/238cadda4c68bfda6794b44402eb67f20cfb3a06/packages/babel-plugin-proposal-decorators/src/index.js#L12-L17).

So here's my thought: any `--plugin-option` argument can be unnamed, and if it is then it just applies to the last `--plugin` provided. Does that sound reasonable? That'd make your README example be this:

```sh
npx @codemod/cli -p import-move-codemod -o @codemod.json --printer prettier /src 
```

**Comment by RIP21** [comment]
Makes total sense! Will add that to Readme :) Thanks a lot. If you'll be able to add such note somewhere to the Readme of the cli that would be awesome to avoid duplicates :)

Thanks again :)

**Comment by eventualbuddha** [comment]
Just in case I wasn't clear, that feature isn't implemented yet. I'll see if I can get to that today.

**Comment by RIP21** [comment]
@eventualbuddha I thought it's a "feature, not a bug" and it's unfixable that's why I suggested adding docs about it. But it's even better if it's possible to fix :)

**Comment by eventualbuddha** [comment]
This should be addressed in `@codemod/cli@2.2.0`.
